pipeline/outputs/dynatrace.md

```diff
@@ -0,0 +1,61 @@
+| Allow_Duplicated_Headers   | Specifies duplicated header use                                                                                                                                                                                                     |  false                                             |
+| host                       | Your Dynatrace environment hostname where {your-environment-id} is your environment ID.                                                                                                                                             | {your-environment-id}.live.dynatrace.com           |
+| port                       | TCP port of your Dynatrace host                                                                                                                                                                                                     | 443                                                |
+| uri                        | Specify the HTTP URI for Dynatrace log ingest API                                                                                                                                                                                   | /api/v2/logs/ingest                                |
+| format                     | The data format to be used in the HTTP request body                                                                                                                                                                                 | json                                               |
+| json_date_format           | Date format standard for JSON                                                                                                                                                                                                       | iso8601                                            |
+| json_date_key              | Fieldname specifying message timestamp                                                                                                                                                                                              | timestamp                                          |
+| tls                        | Specify to use TLS                                                                                                                                                                                                                  | on                                                 |
+| tls.verify                 | TLS verification                                                                                                                                                                                                                    | on                                                 |
```

Comment:
I reformatted the table to help with content display when editing, and fixed some formatting issues.

Also, did you mean to list the header key twice?

```suggestion
## Configuration Parameters

| Key                        | Description | Default |
| -------------------------- | ----------- | ------- |
| header                     | The specific header for content-type. | `Content-Type application/json; charset=utf-8` |
| header                     | The specific header for authorization token, where {your-API-token-here} is the Dynatrace API token with log ingest scope. | Authorization Api-Token {your-API-token-here} |
| Allow_Duplicated_Headers   | Specifies duplicated header use. | `false` |
| host                       | Your Dynatrace environment hostname where {your-environment-id} is your environment ID. | `{your-environment-id}.live.dynatrace.com` |
| port                       | TCP port of your Dynatrace host. | `443` |
| uri                        | Specify the HTTP URI for Dynatrace log ingest API. | `/api/v2/logs/ingest` |
| format                     | The data format to be used in the HTTP request body. | `json` |
| json_date_format           | Date format standard for JSON. | `iso8601` |
| json_date_key              | Fieldname specifying message timestamp. | `timestamp` |
| tls                        | Specify to use TLS. | `on` |
| tls.verify                 | TLS verification. | `on` |
```

Reply:
Thanks for the edits!
Yes the header key should be used twice. Matching the guide here: https://docs.dynatrace.com/docs/shortlink/lma-stream-logs-with-fluent-bit#customize-fluent-bit-configuration